zone/spells.cpp

```diff
@@ -2044,6 +2044,14 @@ bool Mob::SpellFinished(uint16 spell_id, Mob *spell_target, CastingSlot slot, ui
 	if(!IsValidSpell(spell_id))
 		return false;
 
+	//Cazic Touch targets the pet owner instead of the pet when said pet is tanking.
+	if (RuleB(Spells,CazicTouchTargetsPetOwner) && spell_id == 982 && spell_target->HasOwner()) {
```

Comment:
Another magic number here, at least convert 982 to a constant that lives in at the very least `eq_constants.h` if there isn't another place that is appropriate

eg

```cpp
static const uint32 DB_SPELL_CAZIC_TOUCH = 982;
```

Reply:
Also minor nit: spacing between `Spells,Cazic...`